### ISSUE TITLE
Allow all skins to be bough directly

### ISSUE THREAD

**Issue by deferred-impact** [deferred-impact/battledudes-issue-tracker#32]
Chests are a SCAM and i will not buy them. But i wouldn't mind spending gems to buy an illegal rarity skin directly, even if it was some stupid price like 2000 gems. That's still better than spending tons of gems on chests not knowing if you will even get an illegal to begin with, and if you do, which one.

**Comment by Hippo856** [comment]
The point of illegal items is so you waste tons of gems buying chests

**Comment by deferred-impact** [comment]
No way, really? What's next Hippo, will you tell me that the Earth is a spheroid or that it orbits around the Sun?

Obviously the way everything is set up is intentional. But it is, undeniably, a scam (you seem to agree). I think we players should demand this scam be deleted or otherwise addressed.

If it must remain in game and viable, i see the following option for coexistence:

(assuming the average price of an illegal item via legendary chests is X gems)

 - if you want a specific illegal, you can buy it directly, by spending 2*X gems

 - if you want to get all illegals, chests are the way to go, since you will (most likely) spend N*X gems on all of them, where N is the total amount of illegals in the game

**Comment by Gregory-Y** [comment]
Agree. Sucks that some of these are only available in chests. Should apply to all rarities.

**Comment by deferred-impact** [comment]
Setting it as 5min because really, all you'd need to do is to add these skins to the shop. I'd gladly buy them for some stupid price like 1500 gems per skin. Still much better than rolling dice *hoping* to get a decent deal (and also not getting the skin i wanted in the end)
